_posts/2018-11-15-Rust-survey-2018.md

```diff
@@ -0,0 +1,168 @@
+---
+layout: post
+title: "Rust Survey 2018 Results"
+author: The Rust Survey Team
+---
+
+Another year means another Rust survey, and this year marks Rust's third annual survey. This year, the survey launched for the first time in multiple languages. In total **14** languages, in addition to English, were covered.  The results from non-English languages totalled *25% of all responses* and helped pushed the number of responses to a new record of **5991 responses**. Before we begin the analysis, we just want to give a big "thank you!" to all the people who took the time to respond and give us your thoughts. It’s because of your help that Rust will continue to improve year after year.
+
+![Do you use Rust](/images/2018-11-RustSurvey/1-Do_you_use_Rust.png)
+
+Despite having an increase in the number of responses, this year also saw an increase in the percentage of Rust users. Up from last year’s 66.9% Rust users, this year nearly three-quarters of responses were from Rust users.
+
+# Rust Users
+
+## **Time with Rust**
+
+![How long have you worked in Rust](/images/2018-11-RustSurvey/4-How_long_have_you_worked_in_Rust.png)
+
+We’re seeing a steady stream of new users into Rust. At the time of the survey, ~23% of Rust users had been using it for 3 months or less.  Likewise, nearly a quarter of the users have used Rust for at least 2 years.  
+
+![How long did it take to be productive](/images/2018-11-RustSurvey/5-How_long_did_it_take_to_be_productive.png)
+
+Over 40% of Rust users felt productive in Rust in less than a month of use, and over 70% felt productive in their first year. Unfortunately, there is a noticeable struggle among users, as over 22% do not yet feel productive.
+
+![How long have you been unproductive](/images/2018-11-RustSurvey/5a-How_long_have_you_been_unproductive.png)
+
+Looking closer at these users who feel unproductive in Rust, only about 25% are in their first month of use. The challenge here is to find ways to help bridge users to productivity so they don't get stuck.
```

Comment:
Did we happen to ask why folks feel unproductive, specifically?

Reply:
We didn't ask, unfortunately